bin/population/pycbc_population_plots

```diff
@@ -27,19 +27,23 @@ __date__ = "31.10.2017"
 
 import logging
 import argparse
-import pycbc.version
-
+from numpy import logaddexp, log, newaxis, expm1
 import h5py
-import pylab
 import numpy as np
-import scipy.stats as ss
 from matplotlib import cm
+import pylab
+
+import scipy.stats as ss
+
+import pycbc
+import pycbc.version
 from pycbc.population import rates_functions as rf
-from numpy import logaddexp, log, newaxis, expm1
 
 # Parse command line
-parser = argparse.ArgumentParser(description=
-                                      "Plot rate prior and posteriors.")
+parser = argparse.ArgumentParser(
+    description="Plot rate prior and posteriors."
```

Comment:
Use `__doc__` here and put the description in the docstring.

Reply:
used __doc__, the info was already in the docstring though